e2e/tests/playwright-tests/app.cks.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { test, expect } from '@playwright/test';
+import { selectors } from "./helpers/selectors";
+import { userAuthData } from './config';
+import { encryptDecryptProcedure } from "./helpers/operations";
+import * as fs from "fs/promises";
+
+test.describe.configure({ mode: 'serial' });
+
+let fileName;
+
+test.describe('For CKS user', () => {
+	test('Encrypt/decrypt data A with a CKS user X, a CKS user using the JS SDK', async ({ browser }) => {
+		const contextZ = await browser.newContext({ storageState: 'e2e/tests/playwright-tests/.auth-cks/user.json' });
```

Comment:
How does the user info get from the accounts.json file into this one?

Reply:
it'll be in next ticket https://virtru.atlassian.net/browse/PLAT-2540